Changes

```diff
@@ -2,8 +2,11 @@ Revision history for Perl extension App::Sqitch
 
 1.1.1
      - Fixed partitioned search for deployed changes on Oracle and Exasol to
-       correctly support plans with more than 250 changes. Thanks to @Nicqu for
-       the report (#521).
+       correctly support plans with more than 250 changes. Thanks to @Nicqu
+       for the report (#521).
+     - A verbosity level of 3 or above (`-vvv`) now sets trace level one
+       force all registry connections via the DBI (which currently means all
```

Comment:
Looks a missing word? Maybe "to force"? But honestly, the way this is phrased is really confusing to me.

Reply:
Fair point. Rewritten in c5a13bbe.

Reply:
Looks good.